<launch>
	<node pkg="simple_arm_control" type="move_right_arm_multiple_goals" name="move_arm_mutliple_goals" output="screen">
     
     <param name="planner_service_name" value="sbpl_planning/plan_path"/>

	   <param name="number_of_goals" value="2"    type="double"/>

   	   <param name="goal_x" value="0.293"    type="double"/>
	   <param name="goal_y" value="-0.799"  type="double"/>
	   <param name="goal_z" value="0.837"     type="double"/>

	   <param name="goal_roll" value="-2.5"  type="double"/>
	   <param name="goal_yaw" value="1.23"    type="double"/>
	   <param name="goal_pitch" value="1.393"  type="double"/>


	   <param name="goal_x_2" value="0.43"    type="double"/>
	   <param name="goal_y_2" value="-0.42"  type="double"/>
	   <param name="goal_z_2" value="0.9"     type="double"/>

	   <param name="goal_roll_2" value="1.0"  type="double"/>
	   <param name="goal_yaw_2" value="0.5"    type="double"/>
	   <param name="goal_pitch_2" value="0.2"  type="double"/>
     
     <param name="planning_duration" value="20.0" type="double" />

	</node>
</launch>
